src/buildercore/bootstrap.py

```diff
@@ -139,6 +142,7 @@ def is_updating(stackname):
            "Failed to create stack: %s.\nEvents: %s" % (final_stack.stack_status, pformat(events)))
 
 def setup_ec2(stackname, context_ec2):
+    # TODO: misleading name, doesn't set up anything but only waits
```

Comment:
actually it does setup something at the end of the method... ignore this comment.